runtime/tests/checker/interface_test.go

```diff
@@ -3717,244 +3180,6 @@ func TestCheckInterfaceTypeDefinitionInheritance(t *testing.T) {
 
 	t.Parallel()
 
-	t.Run("type requirement", func(t *testing.T) {
```

Comment:
We might want to add some tests for the behavior of interface inheritance with events. Previously they worked the same way as type requirements, but now that they behave differently, existing tests may not cover their new behavior.

Reply:
we could add a test to ensure they are not inherited (if not already exist)

Reply:
I believe the tests in https://github.com/onflow/cadence/pull/2662 already assert that they are not inherited.

Reply:
nice!